test/fuzz/basic.ml

```diff
@@ -477,6 +477,19 @@ let expr_call (env : Env.t) (stack : val_type list) =
       | _ -> None )
     env.funcs
 
+let expr_br_if (env : Env.t) (stack : val_type list) =
+  List.filter_map
+    (fun (name, bt) ->
+      match bt with
+      | Arg.Bt_raw (_, (_pt, rt))
+        when S.is_stack_compatible (List.tl stack) (List.rev rt) ->
```

Comment:
This is going to fail when the stack is empty (of course this should not happen).

You could rewrite this like:

```ocaml
let expr_br_if (env : Env.t) (stack : val_type list) =
  match stack with
  | [] -> []
  | _hd::tl ->
  List.filter_map
    (fun (name, bt) ->
      match bt with
      | Arg.Bt_raw (_, (_pt, rt))
        when S.is_stack_compatible tl (List.rev rt) -> ...
```

This also avoid doing `List.tl` many times.